sentence-embedding/custom-recipes/sentence-embedding-compute/recipe.py

```diff
@@ -69,7 +70,7 @@
         embedded_texts = model.get_weighted_sentence_embedding(texts, smoothing_parameter, npc)
 
     # Checking for existing columns with same name
-    new_column_name = "{}_{}".format(name, aggregation_method)
+    new_column_name = "{}-{}".format(name, aggregation_method)
```

Comment:
sure? It breaks compatibility with previous version of the plugin!

Reply:
this must have been a typo, I changed it back